### PR TITLE
Xavier manual deletion sql

### PR DESCRIPTION
**Pull request by julian1**

### CODE REVIEW

File: scripts/sql/xavier-changes-16-03-2016/aatamsDB_queries.sql [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+WHERE device.id = receiver_deployment.receiver_id 
+AND device.serial_number = '102044' AND initialisationdatetime_timestamp IS NULL;
+
+
+
+
+--------------------
+---- Add 'RANGE TEST' tag as a new type of transmitter in the transmitter_type table. TO DO: update pinger tags which we know are range test tags in the sensor table (Andre to provide list).
+INSERT INTO transmitter_type (id, version, transmitter_type_name) VALUES
+(37, 0, 'RANGE TEST');
```
Comment (jonescc) [inline scripts/sql/xavier-changes-16-03-2016/aatamsDB_queries.sql]: How are these normally inserted?

Comment (julian1) [inline scripts/sql/xavier-changes-16-03-2016/aatamsDB_queries.sql]: I'm not sure. @xhoenner ?

**Comment by jonescc** [comment]
Are these queries or updates? aatamsDB_queries.sql is misleading

**Comment by julian1** [comment]
@jonescc changed filename

File: scripts/sql/xavier-changes-16-03-2016/aatamsDB_queries.sql (hunk 1 above)
Comment (xhoenner) [inline scripts/sql/xavier-changes-16-03-2016/aatamsDB_queries.sql]: I asked this very question to @jkburges when working on this query:
"Just for our records, would you mind advising where the content of the device_model comes from? I saw that in reference_data.sql two receiver models are inserted into that table using insert queries, but can’t find how all the others got in there."

His answer was:
> They were probably inserted with ad-hoc queries, such as the one that you've suggested.
Ideally, there'd be a page in the app for admins to insert device models themselves, but it's one of the things for which there was never time!

**Comment by julian1** [comment]
@jonescc @xhoenner I extracted the materialized view deletion sql. See https://github.com/aodn/aatams/pull/296

File: scripts/sql/xavier-changes-16-03-2016/updates.sql [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+AND device.serial_number = '102044' AND initialisationdatetime_timestamp IS NULL;
+
+
+
+
+--------------------
+---- Add 'RANGE TEST' tag as a new type of transmitter in the transmitter_type table. TO DO: update pinger tags which we know are range test tags in the sensor table (Andre to provide list).
+INSERT INTO transmitter_type (id, version, transmitter_type_name) VALUES
+(37, 0, 'RANGE TEST');
+
```
Comment (jonescc) [inline scripts/sql/xavier-changes-16-03-2016/updates.sql]: Its possible to do this through the UI (show sensor and click on an existing transmitter_type or go directly to aatams/transmitterType/list).  We should use the UI in preference to using sql.

Comment (xhoenner) [inline scripts/sql/xavier-changes-16-03-2016/updates.sql]: Thanks @jonescc, I didn't know it was possible to do so through the UI. You can comment out this query then.

**Comment by xhoenner** [comment]
@astecken you can now update pinger tags which we know are range test tags in the sensor table as we now have a new tag type: 'Range test'.

**Comment by astecken** [comment]
Thanks, will do :)

File: scripts/sql/xavier-changes-16-03-2016/updates.sql [hunk 3]
```diff
@@ -1,4 +1,4 @@
-﻿set search_path = aatams, public;
+﻿set:wq search_path = aatams, public;
```
Comment (julian1) [inline scripts/sql/xavier-changes-16-03-2016/updates.sql]: seduced by the power of vim!

File: scripts/sql/xavier-changes-16-03-2016/updates.sql [hunk 4]
```diff
@@ -33,8 +33,8 @@ AND device.serial_number = '102044' AND initialisationdatetime_timestamp IS NULL
 
 --------------------
 ---- Add 'RANGE TEST' tag as a new type of transmitter in the transmitter_type table. TO DO: update pinger tags which we know are range test tags in the sensor table (Andre to provide list).
-INSERT INTO transmitter_type (id, version, transmitter_type_name) VALUES
-(37, 0, 'RANGE TEST');
+-- INSERT INTO transmitter_type (id, version, transmitter_type_name) VALUES
+-- (37, 0, 'RANGE TEST');
```
Comment (julian1) [inline scripts/sql/xavier-changes-16-03-2016/updates.sql]: would it be better to remove this entirely rather than comment it?

**Comment by julian1** [comment]
Something's gone wrong here. I created the PR, @jonescc had comments. @xhoenner added new code. Now I'm making comments in response.

**Comment by julian1** [comment]
It would probably be better to keep the PRs separate.

File: scripts/sql/xavier-changes-16-03-2016/updates.sql [hunk 5]
```diff
@@ -0,0 +1,62 @@
+﻿set:wq search_path = aatams, public;
```
Comment (julian1) [inline scripts/sql/xavier-changes-16-03-2016/updates.sql]: Trying to run this gives a syntax error.

```
aatams3=# set:wq search_path = aatams, public;
ERROR:  syntax error at or near ":"
LINE 1: set:wq search_path = aatams, public;
```

**Comment by julian1** [comment]
There's a syntax error in the sql. Also should be squashed, and not spread out over four commits.

**Comment by julian1** [comment]
It won't run, so I have reverted here https://github.com/aodn/aatams/pull/298

**Pull request merged**